packages/design-tokens/src/tokens/functional/components/bento/colors.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  brand: {
+    Bento: {
+      background: {
```

Comment:
```suggestion
      bgColor: {
```

😄

Reply:
Updated in this [SHA](https://github.com/primer/brand/pull/390/commits/25fcf6f8e2feb02da8119da3a847cbdf161e7807)